R/leanify.R

```diff
@@ -43,6 +43,9 @@ leanificate_method = function(cls, fname, env = cls$parent_env) {
   environment(replacingfn) = environment(fn)
 
   function_kind = switch(function_kind_container, public_methods = "public", private_methods = "private", active = "active")
+  # otherwise leanification is not applied properly when installing mlr3 with --with-keep.source
+  # this can cause serialized objects to be unreasonably large
```

Comment:
would change the comment to "remove unneeded srcref attributes that lead to (1) incorrect rendering of leanified functions and (2) use up memory unnecessarily" or something like that. I wouldn't call what is happening "not applied properly"